docs/content/guides/developer/sui-101/send-txn.mdx

```diff
@@ -1,6 +1,58 @@
 ---
 title: Sending Transactions
-draft: true
 ---
 
-Content coming soon
\ No newline at end of file
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'; 
+
+### Overview
+In Sui, a transaction is more than a basic record of the flow of assets. Transactions in Sui represent calls to specific functionality (e.g., calling a smart contract function) that execute on inputs to define the result of the transaction.
+Inputs can either be an object reference (either to an owned object, an immutable object, or a shared object), or an encoded value (for example, an encoded string used as an argument to a Move call). 
+Once a transaction is constructed, usually through using [Programmable Transaction Blocks](https://docs.sui.io/guides/developer/sui-101/building-ptb), the user signs the transaction and submits it to be executed on-chain.
+
+### Workflow
+
+The following high-level process describes the overall workflow for constructing and executing a transaction on-chain:
+* The user finds the ObjectID of the gas coin object to be used to pay for gas. If there is no gas coin object, then the user should first use the [splitCoin](https://docs.sui.io/guides/developer/sui-101/building-ptb#available-transactions) transaction to create a gas coin object, that can be done in the next step.
+* The user constructs the transaction data. This is usually done by using [Programmable Transaction Blocks](./building-ptb).
+* The user signs the transaction to generate a [Signature](../../../../concepts/cryptography/transaction-auth/signatures#user-signature).
+* The user submits the transaction to be executed on-chain.
+
+
+### Examples
+
+Here are a few examples on how to transfer SUI in a transaction in different environments and SDKs.
+
+<Tabs groupId="examples-sending-transaction">
```

Comment:
Is there a way to set a default here? If so, it should be the CLI.

Reply:
Added CLI as default!